src/openforms/forms/migrations/0088_enable_mutiple_registration_backends.py

```diff
@@ -0,0 +1,111 @@
+# Generated by Django 3.2.20 on 2023-07-18 14:08
+
+from django.db import migrations, models
+import django.db.models.deletion
+import openforms.registrations.fields
+from openforms.registrations.registry import register
+
+
+def forward(apps, _):
+    Form = apps.get_model("forms", "Form")
+    FormRegistrationBackend = apps.get_model("forms", "FormRegistrationBackend")
+    for form in Form.objects.exclude(
+        models.Q(registration_backend="") | models.Q(registration_backend__isnull=True)
+    ):
+        backend = form.registration_backend
+        # getting the plugin for a human readable name
+        # as form.get_registration_backend_display doesn't exist
+        plugin = register[backend]
```

Comment:
I think this can lead to interesting crashes if people have stale data with custom registration plugins that have since been removed. Perhaps this can be done in a more robust way:

```py
backend_label = register[backend].get_label() if backend in register else "Surprise!"
```